src/Compilers/CSharp/Portable/CodeGen/CodeGenerator.cs

```diff
@@ -139,7 +139,7 @@ private bool IsDebugPlus()
 
         private bool EnablePEVerifyCompat()
```

Comment:
This reads like a mutating method, but is actually flag-returning. How about `IsPEVerifyCompatEnabled`?

Reply:
Someone suggested this name in past CR. I would be ok with renaming this and perhaps other helpers (like "HasValue") to be more matching with their purpose, but not in this change. We can rename/refactor later.

---
In reply to: [146959260](https://github.com/dotnet/roslyn/pull/22772#discussion_r146959260) [](ancestors = 146959260)